### ISSUE TITLE
NW.JS 012.0, WARNING:channel.cc

### ISSUE THREAD

**Issue by merqlove** [nwjs/nw.js#3201]
Hello guys. 
Just tried to upgrade my app for the new NW.JS!

When development toolbar is in enabled state, i'm receiving similar errors on application exit.

    [58812:0309/012536:WARNING:raw_channel_posix.cc(281)] sendmsg/write/writev: Socket is not connected
    [58812:0309/012536:WARNING:channel.cc(553)] Failed to send message to ack remove remote endpoint (local ID 1, remote ID 1)
    [58812:0309/012536:WARNING:channel.cc(320)] RawChannel write error

OS: Mac OSX 10.9.5

**Comment by merqlove** [comment]
Part of my package.json:

```
...
"single-instance": true,
  "chromium-args": "--disable-webgl --disable-3d-apis --disable-plugins --disable-plugins-discovery --disable-xslt --disable-audio-track-processing",
  "dom_storage_quota": 1024,
  "window": {
    "show": false,
    "toolbar": false,
    "title": "SomeTitle",
    "position": "center",
    "width": 1000,
    "height": 800,
    "min_width": 1000,
    "min_height": 800,
    "icon": "images/favicon.png",
    "no-edit-menu": true,
    "resizable": true,
    "always-on-top": false
  }
...
```

**Comment by rogerwang** [comment]
I'll see to fix this in 0.13.0. Thanks.
